app/src/main/java/org/gdg/frisbee/android/api/PlusImageUrlConverter.java

```diff
@@ -0,0 +1,41 @@
+package org.gdg.frisbee.android.api;
+
+import org.gdg.frisbee.android.api.model.plus.ImageInfo;
+import org.gdg.frisbee.android.app.App;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import okhttp3.Interceptor;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class PlusImageUrlConverter implements Interceptor {
```

Comment:
I did this and I am not very proud. This fixed the memory leak in Experts page before. 

We were doing network operation in the Adapter view handling before. That was causing memory leaks. In GDE page, we don't have imageUrl of all the people, we need to get them one by one. We only have their Google+ url. This converts Google+ URL into an image URL. 

We will change that outdated GDE API anyways. And it seems that it stopped working. So we will eventually get rid of this.